srcopsmetrics/cli.py

```diff
@@ -146,38 +182,30 @@ def get_entities_as_list(entities_raw: Optional[str]) -> List[str]:
     "--sli-slo",
     is_flag=True,
     required=False,
-    help="""Launch sli-slo metrics calculation given repositories.""",
+    help="""Launch sli-slo metrics calculation given repositories. Must be used in conjunction with -t""",
 )
 def cli(
     repository: Optional[str],
     organization: Optional[str],
     create_knowledge: bool,
-    process_knowledge: bool,
+    # process_knowledge: bool,
     is_local: bool,
     entities: Optional[str],
-    visualize_statistics: bool,
-    reviewer_reccomender: bool,
+    # visualize_statistics: bool,
+    # reviewer_reccomender: bool,
```

Comment:
basically all of the process_knowledge, visualize and recommender arguments can bre removed, I would only leave `metrics` option because it is more or less relevant to the potential upcoming feature requests